app/views/grade_entry_forms/_react_grades_table.js.jsx.erb

```diff
@@ -185,6 +185,23 @@
       });
     },
     render: function() {
+      // need to filter for each section
+      var sections = <%= raw @sections.to_json %>;
+      if (sections.length > 0) {
+        var section_filters = sections.map(function(section) {
+          var filter = {
+            name: section.name,
+            text: section.name,
+            func: function(group) {
+              return group.section === section.name;
+            }.bind(section)
```

Comment:
Okay, I guess I wasn't clear. If you use `section` instead of `this`, then you don't need to call `bind` on the function.